src/extension.ts

```diff
@@ -607,6 +607,31 @@ function launchMetals(
       }
     );
 
+    registerCommand(`metals.new-scala-worksheet`, async () => {
+      const sendRequest = (args: Array<string | undefined>) => {
+        return client.sendRequest(ExecuteCommandRequest.type, {
+          command: ServerCommands.NewScalaFile,
+          arguments: args,
+        });
+      };
+      const currentUri = window.activeTextEditor?.document.uri;
+      if (currentUri) {
```

Comment:
nitpick, but since `""` is falsy in JavaScript, I usually make these checks explicit, to avoid weird edge-case behaviors:

```suggestion
      if (currentUri != null) {
```

`!= null` will check for both `null` and `undefined` (different than `!== null`, sigh, JS...)

Reply:
Fixed!